pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -529,3 +658,18 @@ func storeIntegrityToken(token []byte) {
 func readIntegrityToken() ([]byte, error) {
 	return ioutil.ReadFile(types.ITokenFile)
 }
+
+func restartAttestation(ctx *zedagentContext) error {
+	log.Info("[ATTEST] Starting attestation task")
+	if ctx.attestCtx == nil {
+		return fmt.Errorf("No attest module context")
+	}
+	if ctx.attestCtx.attestFsmCtx == nil {
+		return fmt.Errorf("No state machine context found")
+	}
+	log.Infof("Creating %s at %s", "attestFsmCtx.EnterEventLoop",
+		agentlog.GetMyStack())
```

Comment:
Why do you need yo start a new goroutine? Do you know the old goroutine terminated?

Reply:
This is stale code, not used anywhere. Got in by mistake. Removed it.